examples/AerosolActivation/aerosol_activation.jl

```diff
@@ -0,0 +1,275 @@
+    initial_par,
+    truth_sample,
+    truth_array.obs_noise_cov, EKP.EnsembleKalmanProcessModule.Inversion(),
+    Δt=1
+)
+
+# Finally, we can run the Ensemble Kalman Process calibration.
+ϕ_n_values = []
+for n in 1:N_iter
+    θ_n = EKP.EnsembleKalmanProcessModule.get_u_final(ekiobj)
+
+    ϕ_n = mapslices(
+        x -> EKP.ParameterDistributionStorage.transform_unconstrained_to_constrained(priors, x),
+        θ_n;
+        dims=1
+    )
+
+    G_n = [run_activation_model(ϕ_n[:, i]...) for i in 1:N_ens]
+
+    G_ens = hcat(G_n...)
+    EKP.EnsembleKalmanProcessModule.update_ensemble!(ekiobj, G_ens)
+
+    global ϕ_n_values = vcat(ϕ_n_values, [ϕ_n])
+end
+
+# We define some simple functions for plotting the data.
+function plot_ensemble_scatter(id)
+
+    ensemble_member = range(1, N_ens, length=N_ens)
+
+    ylabel = "TODO"
+    title = "TODO"
+    if id == 1
+        ylabel = "Molar mass [kg/mol]"
+        title = "molar_mass_scatter.svg"
+    end
+    if id == 2
+        ylabel = "Osmotic coefficient [-]"
+        title = "osmotic_coeff_scatter.svg"
+    end
+
+    plot(ensemble_member, ϕ_n_values[1][id,1:N_ens],
+          seriestype=:scatter, xlabel="Ensemble Number", ylabel=ylabel, leg=false
+         )
+    for it in range(2, N_iter, length=N_iter-1)
+        it = Int(it)
+        plot!(ensemble_member.+((it-1)*50), ϕ_n_values[it][id, 1:N_ens], seriestype=:scatter, leg=false)
+    end
+
+    current()
+    savefig(title)
+end
+
+function plot_ensemble_means(id)
+
+    number_of_iters = range(1, N_iter, length = N_iter)
+    means = zeros(N_iter)
+
+    for it in range(1, N_iter, length = N_iter)
+        it = Int(it)
+        means[it] = mean(ϕ_n_values[it][id, 1:N_ens])
+    end
+
+    ylabel = "TODO"
+    title = "TODO"
+    if id == 1
+        ylabel = "Molar mass [kg/mol]"
+        title = "molar_mas_average.svg"
+    end
+    if id == 2
+        ylabel = "Osmotic coefficient [-]"
+        title = "osmotic_coeff_average.svg"
+    end
+
+    plot(number_of_iters, means, xlabel = "ensemble number", ylabel = ylabel, label = "enseble mean")
+    hline!([default_params[id]], label="true value")
+    savefig(title)
+end;
+
+# We plot the ensemble members and the ensemble mean for the
+# molar mass and osmotic coefficient.
+plot_ensemble_scatter(1)
+plot_ensemble_means(1)
+plot_ensemble_scatter(2)
+plot_ensemble_means(2)
+# ![](molar_mass_scatter.svg)
+# ![](osmotic_coeff_scatter.svg)
+# ![](molar_mass_average.svg)
+# ![](osmotic_coeff_average.svg)
+
+# Finally we test that the parameter values obtained via EKP.jl are close
+# to the known true parameter values.
+molar_mass_ekp = mean(ϕ_n_values[N_iter][1, 1:N_ens])
+osmotic_coeff_ekp = mean(ϕ_n_values[N_iter][2, 1:N_ens])
+
+println(molar_mass_ekp, " vs ", molar_mass_true)
+println(osmotic_coeff_ekp, " vs ", osmotic_coeff_true)
+
+#@test isapprox(molar_mass_ekp, molar_mass_true, rtol = 0.2)
+#@test isapprox(osmotic_coeff_ekp, osmotic_coeff_true, rtol = 0.1)
+
```

Comment:
delete those? definitely `@test` won't work unless you add `using Test` in the beginning and also add `Test` as a dep in the `docs/Project.toml`

Reply:
I thought I had `using Test` in the beginning of the file. But if we want to avoid the additional dependency I can skip on testing the convergence.

Reply:
Deleted the test. Just left the print